.opensource/project.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "Flamelink JS SDK",
+  "platforms": ["Web"],
```

Comment:
What options do we have for `platforms` here? Thinking that this SDK works for server environments as well as web/native platforms like React Native.

Reply:
Web, iOS, Android

Reply:
Okay, thanks. Interesting breakdown.